build.zig

```diff
@@ -22,8 +27,16 @@ pub fn build(b: *std.build.Builder) void {
         const client = b.addExecutable("showdown", "src/client/main.zig");
         client.addPackage(pkgs.network);
         client.addPackage(pkgs.args);
+        client.addPackage(pkgs.pixel_draw);
         client.setTarget(target);
         client.setBuildMode(mode);
+
+        // NOTE(Samuel): This is temporary
+        if (@import("builtin").os.tag == .linux) {
```

Comment:
I think this is better with `!= .windows`

Reply:
ok, aready changed.